validation.py

```
@@ -0,0 +1,681 @@
+                code="too_long"
+            ))
+        
+        return errors
+    
+    def _contains_spam_content(self, text: str) -> bool:
+        """Check if text contains spam patterns."""
+        text_lower = text.lower()
+        for pattern in self.spam_patterns:
+            if pattern.search(text_lower):
+                return True
+        return False
```

Ruff (0.12.2)

Use `return any(pattern.search(text_lower) for pattern in self.spam_patterns)` instead of `for` loop

Replace with `return any(pattern.search(text_lower) for pattern in self.spam_patterns)`

(SIM110)